python/xorbits/_mars/tensor/einsum/tests/test_einsum_execution.py

```diff
@@ -84,3 +84,18 @@ def test_einsum_execution(setup):
         "ajk,kl,jl,a->a", data1, data2, data3, data4, optimize="greedy"
     )
     np.testing.assert_almost_equal(res, expected)
+
+    # test trace
```

Comment:
Could you add a test to check if it works for multiple chunks, like `tensor_nested = tensor(data_nested, chunk_size=3)`.